src/Logger/IBanditLogger.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Eppo\Logger;
+
+interface IBanditLogger extends LoggerInterface
```

Comment:
Allows customers to only implement assignment logging if they're not using bandits.